packages/react-instantsearch-hooks/src/components/__tests__/Index.test.tsx

```diff
@@ -29,9 +30,11 @@ describe('Index', () => {
     const searchClient = createSearchClient({});
 
     const { container } = render(
-      <InstantSearch indexName="indexName" searchClient={searchClient}>
-        <Index indexName="childIndex">Children</Index>
-      </InstantSearch>
+      <StrictMode>
```

Comment:
do the tests pass without strict mode too? Do we need a separate non-strict version of the tests, or can it not cause an issue?

Reply:
They do pass without Strict Mode.

At first I ran each with both regular mode and Strict Mode, but since we expect behaviors to be strictly identical, I figured we better just use Strict Mode.